### DISCUSSION TITLE
Autocompletion while renaming

### DISCUSSION THREAD

**Discussion by SanAntonio37** [jarun/nnn#1690] [Ideas]
Hi.

First of all, congratulations for this marvelous software.

Just an idea : what about autocomplete the filename while renaming a single file inside n3 (based on existing files in the current directory).
This feature is offered by vifm.

 Best regards.

**Comment by jarun** [comment]
Sorry, in general we don't do completions to avoid external dependency (e.g. readline completion).
Also, auto-completion with existing filenames would only create confusion.
You can write a plugin which would invoke external completion support (e.g. bash completion).
